src/components/UserManagement/UserManagement.jsx

```diff
@@ -84,6 +84,7 @@ class UserManagement extends React.PureComponent {
           <React.Fragment>
             {this.popupElements()}
             <UserSearchPanel
+              user={this.props.state.auth.user}
```

Comment:
Have you considered getting this through redux instead? It reduces prop drilling (making it easier to read/modify) and makes the code more consistent.